src/include/miopen/readonlyramdb.hpp

```diff
@@ -55,13 +59,13 @@ class ReadonlyRamDb
 
         auto record = DbRecord{problem};
 
-        if(!record.ParseContents(it->second.content))
+        if(!record.ParseContents(it->second))
         {
             MIOPEN_LOG_E("Error parsing payload under the key: " << problem << " form file "
                                                                  << db_path
                                                                  << "#"
-                                                                 << it->second.line);
-            MIOPEN_LOG_E("Contents: " << it->second.content);
+                                                                 << it->second);
```

Comment:
Is this necessary? Line number in the log is kind of helpful.

Reply:
It is, however, if I create a map from `std::string` -> `CacheItem` the compile time explodes ( more than 30 min and still going). On the contrary a `std::string` -> `std::string` map compiles easily ( less than five mins for a 40k line find and perf db). Hence I removed it.

Reply:
That's weird it takes so long... And I don't think string->string map would provide any nonmarginal performance boost for minutes of compilation it costs. Have you done any measurements on this?

Reply:
I agree that it should not result in runtime performance improvement, my issue is with the length of the compile time. My compile time test is still running, it's been more than an hour now. I am not sure what the reason is, but that is certainly a side effect.

Reply:
The benchmark I was running ( for compile duration) has been running for 103 mins but did not finish.

Reply:
I guess that compile time problem would be resolved if file contents will be stored as a blob.